spec/factories/planning_application.rb

```diff
@@ -57,6 +65,11 @@
     end
   end
 
+  trait :invalidated do
+    status        { :invalidated }
+    determined_at { Time.current }
```

Comment:
Wouldn't an invalidated document have a nil determined_at timestamp?

Reply:
This was a bad copy-paste - thanks for spotting.